Doc/library/functions.rst

```diff
@@ -1726,44 +1726,42 @@ are always available.  They are listed here in alphabetical order.
 
    Example::
 
-      >>> for item in zip(('a', 'b', 'c'), (1, 2, 3)):
+      >>> for item in zip(['a', 'b', 'c'], [1, 2, 3]):
       ...     print(item)
       ...
       ('a', 1)
       ('b', 2)
       ('c', 3)
 
    More formally: :func:`zip` returns an iterator of tuples, where the *i*-th
-   tuple contains the *i*-th element from each of the argument sequences or
-   iterables.
+   tuple contains the *i*-th element from each of the argument iterables.
 
-   :func:`zip` returns a lazy iterable. The elements wouldn't be processed
-   until the iterable is iterated on, e.g. by wrapping in a :class:`tuple`.
+   :func:`zip` is lazy: The elements wouldn't be processed until the iterable
+   is iterated on, e.g. by wrapping in a :class:`list`.
```

Comment:
Maybe also mention a `for` loop as a way to iterate on the iterable? (Obviously we don't need to mention all ways, but using a `for` loop is more idiomatic than wrapping it in `list()`, except at the REPL when you just want to see how it works. :-)

Reply:
Done.